recipes/_master_base_config.rb

```diff
@@ -155,6 +155,14 @@
   not_if { ::File.exist?("/home/#{node['cfncluster']['cfn_cluster_user']}/.ssh/known_hosts") }
 end
 
+# Install scaling.cfg
```

Comment:
A couple of doubts:
* about naming shouldn't we call it in a more specific way, like _aws_scaling_ ?
* is there a reason to not grouping the scaling [and sqswatcher?] configuration under the `/etc/cfncluster` dir ?

Reply:
1. Let me propose a new name: `jobwatcher`, since all it's really doing is watching the queue, I think this is most appropriate.
2. I've added it to `/etc/` since both sqswatcher and nodewatcher are there.